examples/resources/elasticstack_kibana_maintenance_window/resource.tf

```diff
@@ -0,0 +1,27 @@
+provider "elasticstack" {
+  elasticsearch {}
+  kibana {}
+}
+
+resource "elasticstack_kibana_maintenance_window" "my_maintenance_window" {
+  title   = "UPDATE TEST"
+  enabled = true
+
+  custom_schedule {
```

Comment:
This example will not work, the syntax here is to use an =: `custom_schedule = {` (same for the other configs)